data/transition-sites/bis_security.yml

```diff
@@ -1,10 +1,12 @@
 ---
 site: bis_security
-whitehall_slug: department-for-business-innovation-skills
-homepage: https://www.gov.uk/government/organisations/department-for-business-innovation-skills
+whitehall_slug: department-for-business-energy-and-industrial-strategy
+homepage: https://www.gov.uk/government/organisations/department-for-business-energy-and-industrial-strategy
 tna_timestamp: 20201010101010 # Stub timestamp - site is not in TNA
 host: www.securityhealthcheck.bis.gov.uk
-homepage_furl: www.gov.uk/bis
+homepage_furl: www.gov.uk/beis
 global: =301 https://www.gov.uk/government/policies/keeping-the-uk-safe-in-cyberspace
 aliases:
 - securityhealthcheck.bis.gov.uk
+extra_organisation_slugs:
+- department-for-international-trade
```

Comment:
According to the spreadsheet DCMS should be the extra org for this site, not DIT.